### ISSUE TITLE
Incorrect symbol resolve with CALL [ptr]

### ISSUE THREAD

**Issue by wsxarcher** [keystone-engine/keystone#271]
Compiling
```
call [symbol]
ff1513000000
```

With sym resolved as 0x41b000 behave differently from compiling

```
call [0x41b000]
ff1500b04100
```

**Comment by aquynh** [comment]
yes this is a known issue, see https://github.com/keystone-engine/keystone/issues/244. will try to find some time to fix this asap, thanks.

**Comment by wsxarcher** [comment]
The problem was the wrong usage of ctypes pointer. No bug is present in the core.
